RadarSDKTests/RadarAPIHelperMock.h

```diff
@@ -6,9 +6,8 @@
 //
 
 #import <Foundation/Foundation.h>
-#import <RadarSDK/RadarSDK.h>
-
-#import "RadarAPIHelper.h"
+#import <RadarSDK.h>
+#import "../RadarSDK/Internal/RadarAPIHelper.h"
```

Comment:
Will these changes affect how customers add imports in their code as well?

Reply:
No, because only the internal (non-public) headers need to work this way. There's a lot of confusion out there about what should be done for Swift Package Manager support, and this seems to be one approach.